drivers/soundwire/mipi_disco.c

```diff
@@ -88,7 +89,12 @@ int sdw_master_read_prop(struct sdw_bus *bus)
 		}
 	}
 
-	nval = fwnode_property_count_u32(link, "mipi-sdw-supported-clock-gears");
+	scales_prop = "mipi-sdw-supported-clock-scales";
```

Comment:
not following, the helper IS fwnode_property_count_u32().
We need to try the first property name and if not found try the other one.

Reply:
I meant something on top of that. But on the second though it would become an unreadable as we partially hide the property name. So, scratch that idea.